src/net/chat.rs

```diff
@@ -1087,6 +1096,8 @@ impl ConversationP2P {
             StacksMessageType::GetNeighbors => self.handle_getneighbors(peerdb.conn(), local_peer, chain_view, &msg.preamble),
             StacksMessageType::GetBlocksInv(ref get_blocks_inv) => self.handle_getblocksinv(local_peer, burndb, chainstate, chain_view, &msg.preamble, get_blocks_inv),
             StacksMessageType::Blocks(_) => {
+                monitoring::increment_stx_blocks_downloaded_counter();
```

Comment:
This should be a separate counter -- the blocks are being uploaded here, not downloaded.

Reply:
Ha. I'm confused by the next commented line then:
> we can't receive blocks too often, so close this conversation if we do.

Reply:
Right -- this comment is about making sure that the remote node can't DDoS this node by sending it lots of blocks at once.  The `validate_blocks_push()` method will send a `NACK` to this peer and close the socket if the remote peer is has consumed too much bandwidth.

But, it's still important to count the number of blocks pushed to this peer separately from counting the number of blocks requested from this peer :)

Reply:
`rpc.rs` is incrementing another counter - `increment_stx_blocks_served_counter`.
I've renamed `increment_stx_blocks_downloaded_counter` to `increment_stx_blocks_received_counter` to avoid confusion - https://github.com/blockstack/stacks-blockchain/pull/1670/commits/4aa045a525f743ded1e82bd12717bc6faace7d8e